### PR TITLE
RELATED: SD-887 add clearFinished event

### PR DESCRIPTION
**Pull request by lanhq**
## Related PR
gdc-analytical-designer: https://github.com/gooddata/gdc-analytical-designer/pull/3067

### CODE REVIEW

**Comment by yenkins** [comment]
Currently destined for [release R204](https://confluence.intgdc.com/display/Dev/R204).
This release is **zero downtime** release.
Code drop: 2020-05-07 00:00.

**Comment by yenkins** [comment]
Story [SD-887](http://jira.intgdc.com/browse/SD-887) AD - Implement to post events in response to GDC application updated
	Status: Hacking
	Priority: Major
	Assignee: Lan Huynh

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/82/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/165/ : SUCCESS in 53s

File: src/embedding/ad.ts [hunk 1]
```diff
@@ -628,6 +633,35 @@ export namespace EmbeddedAnalyticalDesigner {
...
+    // clear finished
+    //
+
+    /**
+     * This event is emitted when AD successfully performs clear operation.
+     */
+    export type ClearFinished = IADMessageEvent<GdcAdEventType.ClearFinished, IAvailableCommands>;
+
+    /**
+     * Data type of event that was emit after finish clear action
```
Comment (ptquang86) [inline src/embedding/ad.ts:646]: `... emitted ...`

**Comment by naibinh** [comment]
@lanhq , apart from @ptquang86 's comment, it LGTM

**Comment by yenkins** [comment]
Currently destined for [release R205](https://confluence.intgdc.com/display/Dev/R205).
This release is **zero downtime** release.
Code drop: 2020-05-21 00:00.

File: src/embedding/ad.ts (hunk 1 above)
Comment (lanhq) [reply src/embedding/ad.ts]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-check/94/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/176/ : SUCCESS in 48s

**Comment by ptquang86** [comment]
LGTM

**Comment by yenkins** [comment]
## client-libs/gooddata-typings-pull-request-compliance-gate check failed
### require-approval
- pull request must be approved

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/30/ : UNSTABLE in 5s
- gooddata-typings-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-pull-request-compliance-gate/31/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-typings-unit-tests-zuul-docker-el7/177/ : SUCCESS in 25s

**Pull request merged**